SharkCage/CageChooser/CageChooser.cs

```diff
@@ -15,16 +15,20 @@ static class CageChooser
         static void Main()
         {
             // comment out '&& FALSE' to install service on application start
-#if DEBUG && FALSE
```

Comment:
Is this now working without the &&? Maybe then remove the comment above?

Reply:
It was always working, the `&& FALSE` is just for when you don't want to start the install script every time. The comment is the explanation for that but could maybe be improved to explain both with and without `&& FALSE`